charts/substra-backend/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 <!-- towncrier release notes start -->
 
+## [24.6.0] - 2024-04-04
+
+### Changed
```

Comment:
I thought you added towncrier on this changelog too ?

Reply:
Yes but for now the CI check hasn't been changed yet, and there is a question on whether it's useful to use towncrier for manual updates given that we bump the version number at every change.